tools/python/odin_data/meta_writer/meta_writer.py

```diff
@@ -201,11 +202,18 @@ def _create_datasets(self, dataset_size):
         self._logger.debug("%s | Creating datasets", self._name)
 
         for dataset in self._datasets.values():
+            chunks = None
+            try:
+                if None not in dataset.maxshape:
+                    chunks = dataset.maxshape
+            except:
+                if dataset.maxshape is not None:
+                    chunks = dataset.maxshape
```

Comment:
```suggestion
            chunks = dataset.maxshape
            if isinstance(chunks, int):
                chunks = (chunks,)
            if None in chunks:
                chunks = None
```

Sorry I should have made it a multi-line suggestion

Reply:
Instead of catching the error when trying to iterate a scalar, we can just make it a tuple and pass that. `chunks=1` is equivalent to `chunks=(1,)`